server/models.py

```diff
@@ -24,5 +26,20 @@ def __init__(self, *data, **kwargs):
         for key in kwargs:
             setattr(self, key, kwargs[key])
 
-# class Users(dbs.Model):
-#     pass
+
+class User(dbs.Model):
+    id = dbs.Column(dbs.Integer, primary_key=True)
+    username = dbs.Column(dbs.String(32), index=True, unique=True)
+    email = dbs.Column(dbs.String(120), index=True, unique=True)
+    pw_hash = dbs.Column(dbs.String(128))
+    submissions = dbs.relationship(
```

Comment:
Instead of creating an entirely new table for this relationship, you might be able to instead use [`back_populate`](https://docs.sqlalchemy.org/en/latest/orm/relationship_api.html#sqlalchemy.orm.relationship.params.back_populates) for what I'm assuming is supposed to be a [one to many](https://docs.sqlalchemy.org/en/latest/orm/basic_relationships.html#one-to-many) relationship.

Reply:
I think I'm actually doing this. `backref` partially serves the purpose `back_populate` would, except that with `backref` there is no need to explicitly define the relationship in the table (`Submission`) which has as a foreign key the id of another (`User`). I found a [SO answer](https://stackoverflow.com/a/39870250) that communicates what I believe I'm doing here.

Reply:
If I'm still wrong just let me know and I can implement back_populate as a param instead.

Reply:
I just don't think you need a separate table to accomplish this, but if it works, it works!

Reply:
Can you show me what you think this alternative would like instead? My understanding is framed by the explanation in the following documentation: https://docs.sqlalchemy.org/en/latest/orm/backref.html#relationships-backref.

From my understanding, `back_populate` would still require a second table to exist, i.e. Submission.